src/acceptance/helpers/helpers.go

```diff
@@ -274,12 +274,12 @@ func GenerateDynamicScaleOutAndInPolicy(instanceMin, instanceMax int, metricName
 	return string(marshaled)
 }
 
-// GenerateDynamicScaleInPolicyBetween creates a scaling policy, that scales down from 2 instances to 1, if the metric value lies within a given range.
-// Example how the scaling rules must be defined to achieve a "scale down if value is in range" behaviour:
+// GenerateDynamicScaleInPolicyBetween creates a scaling policy that scales down from 2 instances to 1, if the metric value is in a range of [upper, lower].
```

Comment:
**unrelated**

Changed this to also include the upper threshold which is more user friendly.

Range before: `[lower, upper)`
Range after: `[lower, upper]`